daemon/gamemode-dbus.c

```diff
@@ -189,6 +191,84 @@ static int method_query_status_by_pid(sd_bus_message *m, void *userdata,
 	return sd_bus_reply_method_return(m, "i", status);
 }
 
+/**
+ * Handles the RegisterGameByPID D-BUS Method
```

Comment:
Incorrect method name on all of these comments (should be ByPIDFd).

Reply:
Ups. :+1: